### DISCUSSION TITLE
RimRaf is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by junjuners** [warengonzaga/css-text-portrait-builder#17] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/94748574/142729528-ca6a061b-6a3f-426c-b782-90491c037a29.png)
I am getting this issue when running the yarn command.

Thanks.

**Comment by warengonzaga** [comment, marked as answer]
Hi @junjuners, the project uses a module called rimraf or rm -rf use to clean the dist folder.

Based on your issue, rimraf is not installed on your machine meaning you did not fully install all the required modules to run the builder.

To solve this, try running ``yarn install`` first before building your CSS text portrait. I recommend following the [building steps](https://github.com/WarenGonzaga/css-text-portrait-builder#-building-steps) in the readme.

Hope it helps! 👌

**Reply by junjuners** [reply]
okay, i will try it again to see if it works. thaanks. :>

**Reply by warengonzaga** [reply]
yes, let me know if it works, and don't forget to mark the answer to your question if it solves your issue. 😇

**Reply by junjuners** [reply]
![image](https://user-images.githubusercontent.com/94748574/142730208-24202289-6840-4491-984e-a44ca83bea40.png)
when i served it, it gives this to me.

**Reply by warengonzaga** [reply]
it looks like you are using the 1.0.0 version of the project please use the latest version which is currently here.
https://github.com/WarenGonzaga/css-text-portrait-builder/releases

**Reply by junjuners** [reply]
It has worked for meee.

Thanks a bunch! :>
